packages/sdk/src/components/grid/hooks/useKeyboardSelection.ts

```diff
@@ -143,6 +144,17 @@ export const useKeyboardSelection = (props: ISelectionKeyboardProps) => {
     }
   );
 
+  useHotkeys(
+    ['PageUp', 'PageDown'],
+    () => {
+      const delta = coordInstance.containerHeight - coordInstance.rowInitSize - 1;
+      scrollBy(0, isHotkeyPressed('PageUp') ? -delta : delta);
+    },
+    {
+      enabled: !isEditing,
```

Comment:
All grid shortcut keys should be bound to the active cell:
```
enabled: Boolean(activeCell && !isEditing)
```

Reply:
@Sky-FE Don't agree. PageUp/PageDown in my code only scrolls through the view and doesn't change the active cell. Scroll is not locked, so these keys also shouldn't be.


https://github.com/teableio/teable/blob/aeb01a70e9c3ede48945bf447243446ac2ca8236/packages/sdk/src/components/grid/hooks/useKeyboardSelection.ts#L38-L44

As with CTRL+Z, scrolling should work regardless of whether I have anything selected.

Reply:
@kpodp0ra In fact, the shortcut keys like CTRL+Z is for the entire table. It can undo operations related to views, fields, and records. However, the shortcut keys like pageUp/pageDown should be bound to the grid and should not affect inactive grid. In addition, when turning pages, activeCell should follow the changes.